### PR TITLE
[ADD] base_test_chrome

### PR DESCRIPTION
**Pull request by hbrunn**
This addon makes Odoo v12's chrome based tests available for lower versions in order not to constantly have to work around phantomjs quirks.

Should be very simple to forward port to 9, 10, 11

### CODE REVIEW

**Comment by hbrunn** [comment]
@mart-e is the attribution in https://github.com/OCA/server-tools/pull/1643/files#diff-99dd4eacb4161ba180bcbc5f42831f55R71 and https://github.com/OCA/server-tools/pull/1643/files#diff-727577c155200751bc6491d6d9108c86R7 sufficient or do you prefer some other way of doing it?

**Comment by mart-e** [comment]
@hbrunn no that's good, thanks for checking

File: base_test_chrome/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Run JS tests using chrome",
+    "version": "8.0.1.0.0",
+    "author": "Therp BV,Odoo Community Association (OCA),Odoo SA",
+    "license": "LGPL-3",
+    "category": "Tools",
+    "summary": "Use a modern browser for running client side tests",
+    "depends": [
+        'base',
```
Comment (yajo) [inline base_test_chrome/__openerp__.py]:
```suggestion
        'web',
```

Comment (hbrunn) [reply base_test_chrome/__openerp__.py]: thanks, good one

**Pull request closed without merging**